### PR TITLE
Deprecate the Repeated API

### LINKED ISSUE #1295: Deprecate the Repeated API in 5.0.0
Related to #1292, #808

### PR DESCRIPTION
**Pull request by thomaslevesque**
Fixes #1295

### CODE REVIEW

**Comment by thomaslevesque** [comment]
Oops. Looks like I broke something... I'll look into it.

**Comment by thomaslevesque** [comment]
`VerifyCSharpCodeFix`/`VerifyVisualBasicCodeFix` didn't like the warnings due to the usage of obsolete members... fixed.

File: tests/FakeItEasy.Analyzer.CSharp.Tests/RepeatedAssertionAnalyzerTests.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ namespace FakeItEasy.Analyzer.CSharp.Tests
     public class RepeatedAssertionAnalyzerTests : CodeFixVerifier
     {
         private const string CodeTemplate = @"using FakeItEasy;
+#pragma warning disable CS0618
```
Comment (blairconrad) [inline tests/FakeItEasy.Analyzer.CSharp.Tests/RepeatedAssertionAnalyzerTests.cs]: HA! I thought you added  pragma to the code, not a pragma to the test data. How embarrassing. Carry on!

**Comment by thomaslevesque** [comment]
Rebased onto develop. Should be OK to merge now

**Comment by blairconrad** [comment]
> Rebased onto develop. Should be OK to merge now

It was… :wink:

**Comment by thomaslevesque** [comment]
> It was… 😉

Rebased again!

**Pull request merged**